### PR TITLE
[module] read in and decode stereo camera messages

### PR DESCRIPTION
**Pull request by kirkscheper**
Added a module to read in and decode messages from a stereo camera using a UART connection. Module is in sw/airborne/modules/sensors. 

Please let me know if I need to move, change update anything.

### CODE REVIEW

**Comment by kirkscheper** [comment]
Idea... I can also make the baud rate configurable in the airframe file. Might be useful considering the issues with the previous attempt to push to master.

**Comment by gautierhattenberger** [comment]
that is indeed a good idea to be able to set both port and baudrate

**Comment by flixr** [comment]
Would also make sense to not "hardcode" `SEND_STEREO` to true in the makefile...
Better define it to `TRUE` by default in the .c file and check it with `#if SEND_STEREO`
Or even better make it a flag that you can change at runtime...

Also please add the configure and define options to the documentation section in the module xml.

**Comment by flixr** [comment]
And probably most importantly: what kind of "stereocam" is this module for??

**Comment by kirkscheper** [comment]
The stereocam here is a custom design by Christophe de Wagter that is extensively used at the TU Delft. We have a couple of people working on this camera now in paparazzi, that's why I am trying to push this module now.

**Comment by flixr** [comment]
Thx!
Would be really nice if you could document the configure and define options with their respective xml tags in the module xml.
That way they are properly used in the generated documentation.... (see e.g. the digital cam module):

File: sw/airborne/modules/sensors/stereocam.c [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, write to
+ * the Free Software Foundation, 59 Temple Place - Suite 330,
+ * Boston, MA 02111-1307, USA.
+ *
+ */
+
+/** @file modules/stereocam.c
```
Comment (flixr) [inline sw/airborne/modules/sensors/stereocam.c]: path incorrect...

File: sw/airborne/modules/sensors/stereocam.h [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, write to
+ * the Free Software Foundation, 59 Temple Place - Suite 330,
+ * Boston, MA 02111-1307, USA.
+ *
+ */
+
+/** @file modules/stereocam.h
```
Comment (flixr) [inline sw/airborne/modules/sensors/stereocam.h]: same as in c file, please correct to modules/sensors/stereocam.h

File: sw/airborne/modules/sensors/stereocam.h [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  uint8_t *data;
+} uint8array;
+
+extern uint8array stereocam_data;
+
+extern void stereocam_start(void);
+extern void stereocam_stop(void);
+extern void stereocam_periodic(void);
+
+#endif /* LISA_STEREO_H_ */
```
Comment (flixr) [inline sw/airborne/modules/sensors/stereocam.h]: not really important, but since I noticed it: the define is `STEREOCAM_H_` not `LISA_STEREO_H_`

**Comment by flixr** [comment]
Thx!

**Pull request closed without merging**